google-http-client/src/main/java/com/google/api/client/util/StringUtils.java

```diff
@@ -48,13 +41,15 @@
    * @return encoded bytes, or <code>null</code> if the input string was <code>null</code>
    * @throws IllegalStateException Thrown when the charset is missing, which should be never
    *         according the the Java specification.
-   * @see <a href="http://download.oracle.com/javase/1.5.0/docs/api/java/nio/charset/Charset.html"
+   * @see <a href="http://download.oracle.com/javase/7/docs/api/java/nio/charset/Charset.html"
    *      >Standard charsets</a>
-   * @see org.apache.commons.codec.binary.StringUtils#getBytesUtf8(String)
    * @since 1.8
    */
   public static byte[] getBytesUtf8(String string) {
-    return org.apache.commons.codec.binary.StringUtils.getBytesUtf8(string);
+    if (string == null) {
```

Comment:
Would you mind adding a small test for this and newStringUtf8 please?

Reply:
(unless this method is already tested somewhere else?)

Reply:
Added, PTAL.